runtime/onert/core/include/ir/operation/BinaryArithmetic.h

```diff
@@ -27,7 +27,7 @@ namespace ir
 namespace operation
 {
 
-class BinaryArithmetic final : public Operation
+class BinaryArithmetic : public Operation
```

Comment:
@ragmani Is it required for this PR?

Reply:
Yes, it is required but can be split to another PR. Then this PR must wait for merging the new PR.

Reply:
I see. Let's be in this PR.